src/styles/partials/components/_footer.scss

```diff
@@ -0,0 +1,94 @@
+/* Add css to test in this file. It should be cleaned up every pr, as the css should make into the components project.*/
+.dg_county-footer {
+  border-top: $border-width-largest solid $red;
+}
+.dg_secondary-footer {
+  background-color: $gray-dark;
+  padding: $padding-large;
+}
+.dg_copyright {
+  color: $white;
+  font-family: $default-heading-font;
+  font-size: $small-heading-font-size;
+  text-transform: uppercase;
+}
+.dg_download-blurb {
+  font-style: italic;
+}
+.dg_logo-sm-txt {
+  color: $gray;
+  display: block;
+  font-family: $default-heading-font;
+  font-size: $smallest-font-size;
+  font-weight: $font-bold;
+  letter-spacing: 10px;
+  text-transform: uppercase;
+}
+.dg_logo-h1 {
+  color: $black;
+  font-family: $default-heading-font;
+  font-size: $small-heading-font-size;
+  font-weight: $font-bold;
+  text-transform: uppercase;
+}
+.dg_brand-footer {
+  margin-bottom: $default-margin;
+  margin-top: $margin-largest;
+
+  .img-fluid {
+    margin-bottom: $margin-small;
+  }
+
+  p {
+    padding-bottom: $padding-largest;
+  }
+
+  a {
+    text-decoration: none;
+  }
+  .dg_alt-list {
+    &:before {
+      content: "";
+      text-decoration: none;
+    }
+    li {
+      float: left;
+      margin-right: 5%;
+      width: 40%;
+    }
+  }
+  .dg_seal {
+    max-width: 100px;
+  }
+}
+.dg_social-icon_wrapper {
+  margin-bottom: $margin-largest;
+  .dg_icon-link.dg_icon-link--small {
+    i {
+      font-size: $default-icon-size;
+    }
+  }
+}
+
+.dg_navigation-footer {
+  margin-bottom: $default-margin;
+  margin-top: $margin-largest;
+}
+
+.dg_social-footer {
+  margin-bottom: $default-margin;
+  margin-top: $margin-largest;
+}
+
+.dg_lg_gray {
+  color: $gray;
+  font-size: $large-icon-size;
+}
+
+@media (max-width: 767px) {
+  .dg_brand-footer {
+    li {
+      width: 100% !important;
```

Comment:
we do not want to use !important unless we really need to. Does this work without it?

Reply:
Since this overrides a previous width style with the media query, it will need !important to work.